lib/chat/actions.tsx

```
@@ -517,36 +520,45 @@ export const AI = createAI<AIState, UIState>({
-        title,
-        userId,
-        createdAt,
-        messages,
-        path
-      }
+  const session = await auth()
 
-      await saveChat(chat)
-    } else {
-      return
+  if (session && session.user) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)